sql/migrations/1_initial.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE blocks ( 
+	id CHAR(20) PRIMARY KEY,
+	layer INT,
+	validity SMALL INT,
+	block    BLOB
+);
+
+CREATE INDEX blocks_by_layer ON blocks(layer);
+CREATE INDEX blocks_by_validity ON blocks(layer, validity) where validity = 1;
```

Comment:
why only index on value 1? it seems unused. i thought you meant to select blocks where validity == 1 but i don't see that API in sql/blocks/blocks.go.

Reply:
yes i intended to use it, but then i removed methods that used this index. i will drop it